index.js

```diff
@@ -2,7 +2,7 @@ const path = require('path');
 const express = require('express');
 const connectDb = require('./config/db.js');
 require('dotenv').config();
-const port = process.env.PORT || 5000;

```

Comment:
Why you have changed PORT i don't think its related to issue you are working in

Reply:
Oh sorry, I forgot to change the port, Actually I changed the port because the project was not  running on my PC at port 5000.

Reply:
> Oh sorry, I forgot to change the port, Actually I changed the port because the project was not running on my PC at port 5000.

I changed the port to 5000 . 
Please check.